kjourney/src/app/register-user/register-user-model.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { RegisterUserModel } from './register-user-model';
```

Comment:
we need to test this functionality

Reply:
will write some unit tests

Reply:
unable to implement in the timeframe however manual testing passing